src/Voron/Data/RawData/RawDataSmallPageHeader.cs

```diff
@@ -36,5 +30,11 @@ static RawDataSmallPageHeader()
 
         [FieldOffset(14)]
         public ushort PageNumberInSection;
+
+        [FieldOffset(22)]
+        public ulong SectionOwnerHash;
```

Comment:
Future task, when we have the docs for the page headers, need to create a test verifying that we aren't overwriting important fields by accidents by going past the 32 bytes in the header.